frontend/playground-frontend/src/components/Dashboard/Dashboard.js

```diff
@@ -204,6 +293,6 @@ const Dashboard = () => {
 export default Dashboard;
 
 StatusDisplay.propTypes = {
-  statusType: PropTypes.oneOf(["queued", "training", "finished"]),
+  statusType: PropTypes.oneOf(["queued", "STARTING", "SUCCESS"]),
```

Comment:
honestly, just put `PropTypes.string` since the actual string values will be referenced from the backend and we don't want to change many things when needed